orienteer-users/src/main/java/org/orienteer/users/resource/RegistrationResource.java

```diff
@@ -67,7 +73,7 @@ public void writeData(Attributes attributes) throws IOException {
                 if (success) {
                     params.set(PARAMETER_ID, attributes.getParameters().get(PARAMETER_ID).toOptionalString());
                 }
-                RequestCycle.get().setResponsePage(service.getRegistrationPage(), params);
```

Comment:
Is this breaking other usages of orienteer-users?

Reply:
No. We redirect user to login page instead registration page https://github.com/WeaxMe/Orienteer/blob/master/orienteer-users/src/main/java/org/orienteer/users/resource/RegistrationResource.java#L76